clang/tools/clang-offload-wrapper/ClangOffloadWrapper.cpp

```diff
@@ -1883,22 +1770,10 @@ int main(int argc, const char **argv) {
   cl::ParseCommandLineOptions(
       argc, argv,
       "A tool to create a wrapper bitcode for offload target binaries.\n"
-#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
-      "Takes offload target binaries and optional manifest files as input\n"
-#else
       "Takes offload target binaries as input\n"
-#endif // __INTEL_PREVIEW_BREAKING_CHANGES
       "and produces bitcode file containing target binaries packaged as data\n"
       "and initialization code which registers target binaries in the offload\n"
-#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
-      "runtime. Manifest files format and contents are not restricted and are\n"
-      "a subject of agreement between the device compiler and the native\n"
-      "runtime for that device. When present, manifest file name should\n"
-      "immediately follow the corresponding device image filename on the\n"
-      "command line. Options annotating a device binary have effect on all\n"
-#else
       "runtime. Options annotating a device binary have effect on all\n"
-#endif // __INTEL_PREVIEW_BREAKING_CHANGES
       "subsequent input, until redefined.\n"
```

Comment:
Answering my earlier question - we need to reformat this paragraph. I'm not sure what width we're trying to achieve here, but something like this:

```
"A tool to create a wrapper bitcode for offload target binaries.\n"
"Takes offload target binaries as input and produces bitcode file\n"
"containing target binaries packaged as data and initialization code\n"
"which registers target binaries in the offload runtime. Options\n"
"annotating a device binary have effect on all subsequent input,\n"
"until redefined.\n"
```